frontend/src/metabase/browse/components/BrowseModels.tsx

```diff
@@ -144,7 +147,6 @@ export const BrowseModels = () => {
                   error={modelsResult.error}
                   loading={modelsResult.isLoading}
                   style={{ flex: 1 }}
-                  delay={0}
```

Comment:
Hey, just something I was thinking about: without the delays, a an empty `<span>` will be displayed for 300ms. In some cases this might create awkward layout shifts. For example if the page has two large, vertically stacked loading indicators and they (for some reason) use different delays, then the second loading indicator might appear first and then get pushed down the page when the indicator above it appears. I think since the delays are the same with these two components, this won't happen - but I think it's something worth paying attention to. Not now, perhaps, but when polishing things up.